core/commands/repo.go

```diff
@@ -152,3 +158,49 @@ RepoPath        string the path to the repo being currently used
 		},
 	},
 }
+
+var repoFsckCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Removes repo lockfiles",
+		ShortDescription: `
+'ipfs repo fsck' is a plumbing command that will remove repo and level db lockfiles
+`,
+	},
+	Run: func(req cmds.Request, res cmds.Response) {
+
+		configRoot := req.InvocContext().ConfigRoot
+
+		// acquire the repo lock before removing it
+		repo, err := fsrepo.Open(configRoot)
+		defer repo.Close()
+		if err != nil {
+			res.SetError(err, cmds.ErrNormal)
+			return
+		}
+
+		//TODO: acquire levelDB lock before removing it??
+
+		dsPath, err := config.DataStorePath(configRoot)
+		if err != nil {
+			res.SetError(err, cmds.ErrNormal)
+			return
+		}
+
+		dsLockFile := filepath.Join(dsPath, "LOCK") // TODO: get this lockfile programmatically
+		repoLockFile := filepath.Join(configRoot, lockfile.LockFile)
+
+		log.Infof("Removing repo lockfile: %s", repoLockFile)
+		log.Infof("Removing datastore lockfile: %s", dsLockFile)
+
+		os.Remove(repoLockFile)
+		os.Remove(dsLockFile)
+
+		s := fmt.Sprintf("Lockfiles have been removed.")
+		log.Info(s)
```

Comment:
ah, then just remove the `fmt.Sprintf` wrapper. Its not necessary, `s := "Lockfiles...."` works